frontend/packages/shipwright-plugin/src/components/build-details/BuildSpecSection.tsx

```diff
@@ -29,12 +35,18 @@ const BuildSpecSection: React.FC<BuildSpecSectionProps> = ({ obj, buildSpec, pat
         <DetailsItem label={t('shipwright-plugin~Strategy')} obj={obj} path={`${path}.strategy`}>
           {buildSpec.strategy.kind === 'ClusterBuildStrategy' ? (
             <ResourceLink
-              groupVersionKind={getGroupVersionKindForModel(ClusterBuildStrategyModel)}
+              groupVersionKind={getGroupVersionKindForModel(
+                isV1Alpha1Resource(obj)
+                  ? ClusterBuildStrategyModelV1Alpha1
+                  : ClusterBuildStrategyModel,
+              )}
               name={buildSpec.strategy.name}
             />
           ) : (
             <ResourceLink
-              groupVersionKind={getGroupVersionKindForModel(BuildStrategyModel)}
+              groupVersionKind={getGroupVersionKindForModel(
+                isV1Alpha1Resource(obj) ? BuildStrategyModelV1Alpha1 : BuildStrategyModel,
+              )}
```

Comment:
Below this is code that shows the Source URL. This code needs to be updated as well because the Git URL is now saved in source.git.url instead of source.url.